### ISSUE TITLE
Attachment prompt

### ISSUE THREAD

**Issue by elena20ruiz** [MicrosoftDocs/bot-docs#554]
There is an example o library documentation about the AttachmentPrompt? It is mentioned in this page but not explained or linked to more explained documentation. Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 72538432-fde5-22ab-7997-8a34b11aa24e
* Version Independent ID: c469fa5b-2590-ee16-1491-51b90c069204
* Content: [Gather user input using a dialog prompt - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-prompts?view=azure-bot-service-4.0&tabs=javascript)
* Content Source: [articles/v4sdk/bot-builder-prompts.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/articles/v4sdk/bot-builder-prompts.md)
* Service: **bot-service**
* Product: **bot-framework**
* GitHub Login: @JonathanFingold
* Microsoft Alias: **v-jofing**

**Comment by JonathanFingold** [comment]
Many channels allow the user to add one or more attachments to a message. Use this prompt to ask the user to do so. If you're using prompts within a dialog, the _step context's_ `result` property in the following step will contain an array of `Attachment` objects.

Here's the class definition of an attachment, from the repo's botbuilder-js/libraries/botframework-schema/src/**index.ts** file:

https://github.com/Microsoft/botbuilder-js/blob/90ff3567d36bbc9d59b765a36acf6eced2ffe116/libraries/botframework-schema/src/index.ts#L243-L271

**Comment by andrewchungxam** [comment]
Hi there - does this attachment prompt work in C#?

ie. if in a dialog I were to be asking for a TextPrompt?
_dialogSet.Add(new TextPrompt("promptInBot")); //this was working correctly

And then switched to an AttachmentPrompt
_dialogSet.Add(new AttachmentPrompt("attachmentPromptInBot")); //the emulator is not recognizing the attachment  (ie I try to upload attachment and nothing happens).  FYI: I was able to get attachments recognized when I did not use dialogs -- so I'm wondering if AttachmentPrompt is working correctly or if I'm not using it correctly.
